DashWallet/Sources/UI/Explore Dash/Views/DWExploreTestnetContentsView.m

```diff
@@ -245,4 +256,62 @@ - (NSString *)subtitle {
 - (void)setSubtitle:(NSString *)subtitle {
     self.descLabel.text = subtitle;
 }
+
+- (void)addContent:(UIView *)view {
+    UIView *last = [[_contentStack arrangedSubviews] lastObject];
+    [_contentStack setCustomSpacing:10 afterView:last];
+    [_contentStack addArrangedSubview:view];
+    _contentStack.layoutMargins = UIEdgeInsetsMake(0, 0, 10, 0);
+    _contentStack.layoutMarginsRelativeArrangement = YES;
+}
+@end
+
+
+@implementation DWExploreCrowdNodeContentsViewCell : DWExploreTestnetContentsViewCell
+- (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier {
+    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
+    if (self) {
+        [self addCrowdNodeAPYLabel];
+    }
+    return self;
+}
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self addCrowdNodeAPYLabel];
+    }
+
+    return self;
+}
+
+- (void)addCrowdNodeAPYLabel {
+    UIColor *systemGreen = [UIColor colorWithRed:98.0 / 255.0 green:182.0 / 255.0 blue:125.0 / 255.0 alpha:1.0];
```

Comment:
This color is also used in the Tax explainer dialog for the `income` highlight, and possibly other places - it is part of our color system (`system green` color). Maybe we can put it somewhere for reusability?

Reply:
I agree, we have to update our color system with new color and use it.